yarn.lock

```diff
@@ -2035,6 +2036,7 @@ __metadata:
     intersection-observer: ^0.12.2
     jest: ^29.0.3
     jest-environment-jsdom: ^29.0.3
+    jest-preset-stylelint: ^6.0.0
```

Comment:
This one is required for the test to run, but, like `dedent` and `stylelint`, this should really be in dev dependencies.